src/cljdoc/render/meta.clj

```diff
@@ -17,6 +17,7 @@
             [:tbody.lh-copy
              (shortcut "⌘ K" "Jump to recently viewed docs")
              (shortcut "←" "Move to previous page")
-             (shortcut "→" "Move to next page")]]]]]]
+             (shortcut "→" "Move to next page")
+             (shortcut "⌘ /" "Focus the search field")]]]]]]
```

Comment:
Just the docset search right now but I don't see why it couldn't also work for the libraries search (eventually).

Interesting point about focus. "Go to" feels strange to me because it sounds like you're navigating there (see previous shortcuts). Maybe use "Jump to" like the first shortcut?

Reply:
Ah, see, I didn't know what it did! I think I am maybe being nit-picky here. Whatever you think is good works for me!